spec/integration/sequel/record_is_created_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'support/sequel_models/book'
+
+describe 'When sequel record is created' do
```

Comment:
This is an example of work I would like to continue in a future commit, now that the active model classes are all separated I thought it would be beneficial to design integration specs around behavior.

Especially since a lot of tests in `integration_spec.rb` are not integration specs at all.